todo/src/App.js

```diff
@@ -1,18 +1,12 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
```

Comment:
Removing unnecessary items, always good! 👍 And you're keeping consistency by changing `''` to`""`, nice job!